opentelemetry/src/trace/mod.rs

```diff
@@ -200,6 +200,10 @@ pub enum TraceError {
     #[error("Exporting timed out after {} seconds", .0.as_secs())]
     ExportTimedOut(time::Duration),
 
+    /// already shutdown error
+    #[error("{0} already shutdown")]
+    AlreadyShutdown(String),
```

Comment:
Do we expect to use this variant for anything other than `TracerProvider`?

Reply:
Yes, thought to use them for the processors and exporters too. But I believe we can customize it later if required. For now, made it static for TracerProvider.